File: spectra-serve-platform/src/main/java/com/yangxj96/spectra/server/platform/Application.java
package com.yangxj96.spectra.server.platform;

import org.apache.dubbo.config.spring.context.annotation.EnableDubbo;
import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

/**
 * 平台内容的主启动类
 */
@EnableDubbo(scanBasePackages = "com.yangxj96.saas.server.platform.dubbo")
@MapperScan("com.yangxj96.saas.server.platform.mapper")
@EnableDiscoveryClient
@SpringBootApplication
public class Application {
    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

File: yangxj96-serve/yangxj96-serve-platform/src/main/java/com/yangxj96/saas/server/platform/Application.kt
package com.yangxj96.saas.server.platform

import org.apache.dubbo.config.spring.context.annotation.EnableDubbo
import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

/**
 * 平台内容的主启动类
 */
@EnableDubbo
@MapperScan("com.yangxj96.saas.server.platform.mapper")
@EnableDiscoveryClient
@SpringBootApplication
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}